spec/rspec/rails/example/view_example_group_spec.rb

```diff
@@ -90,6 +90,21 @@ module ::ApplicationHelper; end
       end
     end
 
+    describe "routes helpers collides with aset helpers" do
+      it 'delegates to the controller' do
+        with_isolated_config do
+          ::Rails.application.routes.draw { resources :videos }
```

Comment:
I was playing around with this, tried to move this to `before` block to clean it up and spec just failed. I can look into this to make sure it was set back to whatever it was.